src/commands/new.js

```diff
@@ -77,91 +81,136 @@ class NewCommand extends Command {
         if (entry.type == 'Directory') {
           fs.mkdirSync(entryPath, {
             recursive: true
-          })
+          });
         } else {
-          entry.stream()
-            .pipe(fs.createWriteStream(entryPath))
-            .on('error', (error) => {
-              console.error('Could not create Formidablejs project');
+          if (entry.path.split('/').pop() !== 'package-lock.json') {
+            entry.stream()
+              .pipe(fs.createWriteStream(entryPath))
+              .on('error', (error) => {
+                console.error('Could not create Formidablejs project');
 
-              this.exit;
-            })
+                this.exit;
+              });
+
+            console.log(chalk.green('CREATE') + ' ' + entryPath);
+          }
         }
       });
 
-      cli.action.start('Installing dependencies');
+      if (!flags.manager) {
+        const res = await inquirer.prompt([{
+          name: 'manager',
+          message: 'Which package manager do you want to use?',
+          type: 'list',
+          choices: [{name:'npm'}, {name: 'yarn'}],
+        }]);
+
+        flags.manager = res.manager;
+      }
 
-      installDependencies(location)
+      cli.action.start('Installation in progress, this may take a while ☕');
+
+      installDependencies(flags.manager, location, args.name)
     }).catch(() => {
-      cli.action.stop('Could not fetch formidablejs');
+      console.log('Could not scaffold your application');
     });
   }
 };
 
 NewCommand.description = `Craft a new Formidable application`;
 
-const installDependencies = (location) => {
-  const install = exec('npm i --legacy-peer-deps', {
+const installDependencies = (manager, location, name) => {
+  const install = exec(manager == 'npm' ? 'npm i --legacy-peer-deps' : 'yarn install --legacy-peer-deps', {
     cwd: location
   });
 
-  install.stdout.pipe(process.stdout);
+  let failed = false;
+
+  install.stderr.on('data', (data) => {
+    failed = true;
```

Comment:
I think npm warnings are received for stderr. If I am right, you will stop the process for a warning